Flyte Scribe/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./dist/"
+  }
+}
```

Comment:
https://docs.premid.app/en/dev/presence#filling-in-the-tsconfigjson-file

Reply:
Where is the Issue?

Reply:
outdir

Reply:
Fixed outDir issue in https://github.com/PreMiD/Presences/pull/1511/commits/461c43ae697bbbbe89b9dadc4990f93c039e5b50